app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CommentsController < ApplicationController
+  def create
+    @blog = Blog.find(params[:comment][:blog_id])
+    comment = current_user.comments.build(comment_params)
+    if comment.save
+      ContactMailer.comment_mail(@blog).deliver
+      redirect_to blog_path(@blog)
+    else
+      render 'blogs/show'
+    end
+  end
+
+  def destroy
+    comment = Comment.find(params[:id])
```

Comment:
（https://github.com/Tephone/mini_blog4/pull/3/files#r671936414 のコメントと同じ）
```suggestion
    comment = current_user.comments.find(params[:id])
```
で十分です。これだけで `if` は不要です。
LikesControllerではそうしてますよね。

Reply:
たしかにそもそもにuser_id含んでしませば不要でした
書き換えます